noir-projects/noir-protocol-circuits/crates/rollup-lib/src/base/base_rollup_inputs.nr

```diff
@@ -48,8 +45,6 @@ struct BaseRollupInputs {
     sorted_public_data_writes_indexes: [u64; MAX_PUBLIC_DATA_UPDATE_REQUESTS_PER_TX],
     low_public_data_writes_preimages: [PublicDataTreeLeafPreimage; MAX_PUBLIC_DATA_UPDATE_REQUESTS_PER_TX],
     low_public_data_writes_witnesses: [PublicDataMembershipWitness; MAX_PUBLIC_DATA_UPDATE_REQUESTS_PER_TX],
-    public_data_reads_preimages: [PublicDataTreeLeafPreimage; MAX_PUBLIC_DATA_READS_PER_TX],
```

Comment:
Remove it for now because we are not checking it here. And this will be checked in public tail.